<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ThankYou</class>
 <widget class="QMainWindow" name="ThankYou">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(136, 202, 94);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>300</x>
       <y>10</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(136, 202, 94);
font: 75 6pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>Â© 2024 LMMN, Inc. All rights reserved</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>60</y>
      <width>451</width>
      <height>111</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
font: 75 100pt &quot;MS Sans Serif&quot;;
</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Resources/images/SMART.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>160</y>
      <width>461</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Resources/images/Pharmakeutical Kiosk.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>220</y>
      <width>451</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Streamlined Over-the-Counter Medicine Dispensing Innovation
</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Thank you for using</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>390</y>
      <width>461</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 12pt &quot;MS Sans Serif&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Please don't forget to collect you medicine and change on the bin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="back_Button">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>20</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;MS Sans Serif&quot;;
border-radius: 10px; 
background-color: rgb(255, 255, 255);
color: rgb(136, 202, 94);</string>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>471</width>
      <height>141</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>Resources/images/thankyou_people.png</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="waterButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 15pt &quot;MS Sans Serif&quot;;
border-radius: 10px; 
background-color: rgb(255, 255, 255);
color: rgb(136, 202, 94);</string>
    </property>
    <property name="text">
     <string>Click me for water</string>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>label_3</zorder>
   <zorder>frame</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>back_Button</zorder>
   <zorder>label_6</zorder>
   <zorder>label_2</zorder>
   <zorder>waterButton</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
